### PR TITLE
Add new alias property delivery

### PR DESCRIPTION
**Pull request by gabrielfs7**
A new alias (AssessmentProjectId) property for delivery was added:

https://oat-sa.atlassian.net/browse/CON-188

### CODE REVIEW

File: migrations/Version202012151638472348_taoDeliveryRdf.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    public function up(Schema $schema): void
+    {
+        OntologyUpdater::syncModels();
+    }
+
+    public function down(Schema $schema): void
+    {
+        OntologyUpdater::syncModels();
```
Comment (boajer) [inline migrations/Version202012151638472348_taoDeliveryRdf.php:42]: That will not do anything, if `install/ontology/taodelivery.rdf` stay unchanged.

Comment (gabrielfs7) [reply migrations/Version202012151638472348_taoDeliveryRdf.php]: I cannot say you are wrong :)

File: migrations/Version202012151638472348_taoDeliveryRdf.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?php
+/**
```
Comment (bartlomiejmarszal) [inline migrations/Version202012151638472348_taoDeliveryRdf.php:2]:
```suggestion

/**
```

**Review by bartlomiejmarszal — APPROVED** [review]
LGTM

- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by shpran — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**